src/utils/get.ts

```diff
@@ -0,0 +1,10 @@
+
+export function get(obj: Record<string, any> | undefined, path: string, defValue?: any): any {
```

Comment:
I'd like to see a simple unit test for this. You can create a new file `find.test.ts` into the same folder and Jest should pick it up automatically.

Not that we had much unit tests before :) But let's use the "boy scout rule" of _leave the place in better condition you found it_.